src/measurements/ad-blocker.js

```diff
@@ -0,0 +1,49 @@
+import { AddonManager } from 'resource://gre/modules/AddonManager.jsm';
+import BaseMeasurement from '../lib/measurement';
+
+// This list is constructed by manually-reviewing all add-ons in the AMO
```

Comment:
Random thought: I wonder if we might be able to solicit any add-ons we're missing via Test Pilot surveys or at least a note in the README in this repo?